src/Components/Header-Cmd.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import {Link ,Navigate,useNavigate} from 'react-router-dom';
```

Comment:
Hey, please remove Navigate & useNavigate here, as they are unused variables.

```suggestion
import {Link} from 'react-router-dom';
```